packages/ipfs/src/core/components/dag/resolve.js

```diff
@@ -1,16 +1,45 @@
 'use strict'
 
+const CID = require('cids')
 const { parseArgs } = require('./utils')
 const { withTimeoutOption } = require('../../utils')
 
 module.exports = ({ ipld, preload }) => {
-  return withTimeoutOption(async function * resolve (cid, path, options) { // eslint-disable-line require-await
+  return withTimeoutOption(async function resolve (cid, path, options) { // eslint-disable-line require-await
```

Comment:
This function now includes an `await` so `eslint-disable-line require-await` is not necessary.

Reply:
Removed it.